guides/auth-torii-with-github.md

```diff
@@ -31,7 +31,7 @@ configuration and base classes. For GitHub specifically, the authentication flow
    <access_token>` header request to, say, `https://api.github.com/user`,
    which returns the GitHub user’s user info, at which point it can be matched to the app's own `User` model. 
 
-You can find a sequence diagram for the full flow [here](./assets/esa-initial-flow.txt.png).
+You can find a sequence diagram for the full flow [here](./assets/esa-initial-flow.svg).
```

Comment:
Won't this be a broken link as the [file](https://github.com/simplabs/ember-simple-auth/tree/23981b7e39d137bd63440bdc114bcf7daa832ef8/guides/assets) was not generated manually?

Reply:
🤦 fixed